src/sentry/api/endpoints/organization_codeowners_request.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.utils.translation import ugettext as _
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
+
+from sentry import roles
+from sentry.api.bases.organization_request_change import OrganizationRequestChangeEndpoint
+from sentry.models import OrganizationMember, Project
+from sentry.utils.email import MessageBuilder
+from sentry.utils.http import absolute_uri
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationCodeOwnersRequestEndpoint(OrganizationRequestChangeEndpoint):
+    def post(self, request, organization):
+        """
+        Add a invite request to Organization
+        ````````````````````````````````````
+
+        Creates an invite request given an email and suggested role / teams.
+
+        :pparam string organization_slug: the slug of the organization the member will belong to
+        :param string projectId: the id of the project
+
+        :auth: required
+        """
+        if not request.data.get("projectId"):
+            raise ValidationError(
+                "Missing projectId body parameter.", code=status.HTTP_400_BAD_REQUEST
+            )
+        try:
+            project = Project.objects.get(id=request.data["projectId"], organization=organization)
```

Comment:
@NisanthanNanthakumar If this is always tired to single project, why isn't this a project endpoint instead of an organization endpoint? It will automatically handle looking up the project.